content/blog/why-is-redfish®-different-from-other-rest-apis-part-1.md

```diff
@@ -0,0 +1,134 @@
+
+Proprietary monitoring applications (i.e., <a href="https://www.hpe.com/us/en/hpe-greenlake-compute-ops-management.html" target="_blank">HPE Compute Ops Management</a>) or open source applications (<a href="https://github.com/ODIM-Project/ODIM" target="_blank">ODIM</a>, <a href="https://github.com/nsfcac/Nagios-Redfish-API-Integration" target="_blank">Nagios</a>, etc.) constitute native Redfish® clients or can become so using specific plugins.
+
+### Remote or local management
+
+In most cases, Redfish® clients performs <a href="https://en.wikipedia.org/wiki/Out-of-band_management" target="_blank">out-of-band management</a>. However, it is entirely possible to install the client in the OS of the server to be managed and to access the BMC via an internal path to the server (in-band-management). This path can be proprietary like HPE's CHIF (<a href="https://developer.hpe.com/blog/chif-driver-not-found/" target="_blank">Channel Interface</a>) or the <a href="https://en.wikipedia.org/wiki/Ethernet_over_USB" target="_blank">Ethernet over USB</a> protocol which allows an IP connection between the OS and the BMC.
+
+### Protocol and data model separation
+
+The Redfish® protocol specification is published in DMTF document <a href="https://www.dmtf.org/dsp/DSP0266" target="_blank">DSP0266</a> while data modeling in <a href="https://www.dmtf.org/dsp/DSP0268" target="_blank">DSP0268</a>. This separation allows flexibility with regard to the management of version numbers. Without this separation, a minor change to the protocol specifications (e.g. adding an acronym to the glossary) would involve an increment of the overall specification version number and impose a set of unnecessary tests to requalify clients.
+
+#### What’s inside the protocol specification?
+
+* Boring but necessary topics like the normative framework (definitions, symbols, glossary, typographical conventions, etc.).
+* Things a little more interesting like compliance with other standards (<a href="https://www.odata.org/" target="_blank">Open Data</a>, <a href="https://swagger.io/specification/" target="_blank">Swagger/OpenAPI</a>, etc.).
+* We also find very practical things like the ways to authenticate (basic authentication, session token, etc.) and the endpoints where authentication is necessary (everywhere except at the root `/redfish/v1`).
+* Surprising things like receiving an "error" messages for successful requests. Figure 2 shows the successful erasure of a storage logical volume (LUN) whose response contains the `error{}` object with a message explaining the success of the operation! This type of incongruous response is a consequence of compliance with the <a href="https://www.odata.org/" target="_blank">Open Data</a> standard.
```

Comment:
Change "messages" to "message". (since you placed "an" in front of "error", you make it singular)